Kitodo-DataManagement/src/main/java/org/kitodo/data/datafiles/legacyadapters/UghImplementation.java

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Like I have mentioned already before the whole idea of interface is to disable access to UGH in other way than by interface. It has to be moved to API module. If it stays here everyone can still fully access all classes in UGH without using of interfaces.

Reply:
If we move it to the API package, then the API package must have a dependency to the UGH, and that means that still the code can access the UGH by referencing the API package. What you want is good from the idea, but it seems technically not possible to me. This requires some discipline: Just, **nobody use the UGH any more!** It is an old, soul-destroying monster that we are close to condemn forever.

Reply:
It is our goal. Any access should be made only by API. No dependency inside Data Management and Core,